update.sh

```diff
@@ -16,7 +16,11 @@ for latest in "${latests[@]}"; do
 	for variant in apache fpm; do
 		# Create the version+variant directory with a Dockerfile.
 		mkdir -p "$version/$variant"
-		cp Dockerfile.template "$version/$variant/Dockerfile"
+		if [ "$version" == "11.0" ]; then
```

Comment:
This is not really a future-compatible change. I would recommend defining a function like this:

```
# version_gte A B returns whether A >= B
function version_gte() {
	[[ "$(printf '%s\n' "$@" | sort -V | head -n1)" != "$1" || "$1" == "$2" ]];
}
```

Which you could then use like this:

```
template="Dockerfile.template"
if version_gte "$version" "11.0"; then
    template="Dockerfile-php7.template"
fi
cp "$template" "$version/$variant/Dockerfile"
```